emission/net/api/cfc_webapp.py

```diff
@@ -263,6 +263,16 @@ def getSuggestion():
   logging.debug("Output of ret_dir = %s" % ret_dir)
   return ret_dir
 
+@post('/suggestion_sys')
+def getSingleTripSuggestion(tripid):
```

Comment:
Does this work? Don't you need a `<tripid>` parameter or read the tripid from the request json?
Can you show me the logs that indicate that this function is being called instead of `def getSuggestion():`?
Isn't the path for both of them the same (`/suggestion_sys`)?